controllers/spaceprovisionerconfig/spaceprovisionerconfig_controller_test.go

```diff
@@ -102,6 +145,54 @@ func TestSpaceProvisionerConfigValidation(t *testing.T) {
 		assert.NoError(t, reconcileErr)
 		AssertThat(t, spc, Is(NotReadyWithReason(toolchainv1alpha1.SpaceProvisionerConfigToolchainClusterNotFoundReason)))
 	})
+	t.Run("becomes not ready when the referenced ToolchainCluster is marked as not ready", func(t *testing.T) {
```

Comment:
similarly should this be a subtest of "is ready when existing ready ToolchainCluster is referenced" so that we first set the SPC to ready and we verify that the controller put's it back to non ready ?